spec/features/users/sign_out_spec.rb

```diff
@@ -1,13 +1,10 @@
-# Feature: Sign out
-#   As a user
-#   I want to sign out
-#   So I can protect my account from unauthorized access
-feature 'Sign out', :devise do
-
-  # Scenario: User signs out successfully
-  #   Given I am signed in
-  #   When I sign out
-  #   Then I see a signed out message
+feature 'Sign out', %{
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.